### DISCUSSION TITLE
issue in runnning zitadel aplication in local

### DISCUSSION THREAD

**Discussion by prajwalpamin** [zitadel/zitadel#9056] [Q&A]
Im trying to run application in local but when i clone aplication i get error for few unimplemented methods and imports and so on . even after running go mod tidy and go mod download command im unable to solve it .

**Comment by sampath24-ss** [comment]
Try to use docker for running the application locally.

**Comment by fforootd** [comment]
I guess you want to build Zitadel on your own, right?

There is a guide here to do that https://github.com/zitadel/zitadel/blob/main/CONTRIBUTING.md#developing-zitadel-with-dev-containers

To me it sounds like the gRPC stubs are missing and need to be built.

**Comment by sampath24-ss** [comment]
While testing the ./console locally i am facing issue as the page is loading for a long time. Please find the attachment for the reference. 

![Screenshot 2024-12-19 205047](https://github.com/user-attachments/assets/c0cbe4cb-365b-4f6b-8d97-ab45ad6245a9)

**Reply by fforootd** [reply]
How did you start Zitadel?
